AstSemantics.md

```diff
@@ -76,10 +76,8 @@ of bytes. The linear memory is sandboxed; it does not alias the execution
 engine's internal data structures, the execution stack, local variables, global
 variables, or other process memory.
 
-In the MVP, linear memory is not shared between threads. When
-[threads](PostMVP.md#threads) are added as a feature, regular load and store
-nodes will have the most relaxed semantics specified in the memory model and new
-memory-access nodes will be added with atomic and ordering guarantees.
+In the MVP, linear memory is not shared between threads of execution. This will
+change when [threads](PostMVP.md#threads) are added.
```

Comment:
I think this is technically correct, but only in the way that `false implies P` is vacuously true so I think it'd be more clear to reword.  The fact that you can have separate wasm modules executing in separate workers (and thus on separate threads) is irrelevant since the wasm spec only considers the execution of a single module (much like the C spec is only talking about what's happening in a single process).  I guess the goal is to provide a link from here to PostMVP.md#threads; perhaps clearer to just say "When [threads] are added, the memory model will be augmented to define the semantics of concurrent linear heap accesses." or something.